components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/java/org/wso2/carbon/idp/mgt/ui/util/IdPManagementUIUtil.java

```diff
@@ -107,6 +107,20 @@ public static boolean validateURI(String uriString) {
         return true;
     }
 
+    /**
+     * Validates if Scopes have not been configured in both Scopes and Additional Query Parameters.
+     *
+     * @param paramMap Map<String, String>
+     * @throws Exception
+     */
+    private static void validateScopes(Map<String, String> paramMap) throws Exception {
+
+        if (StringUtils.isNotBlank(paramMap.get("scopes")) &&
+                paramMap.get("oidcQueryParam").toLowerCase().contains("scope=")) {
+            throw new Exception("Cannot set scopes in both Scopes and Additional Query Parameters");
```

Comment:
Shouldn't this also say to use the 'scopes' field to set the scopes (ie. to avoid setting scopes in additional query params).